$MirOS: ports/www/firesomething/patches/patch-security_nss_lib_freebl_Makefile,v 1.6 2009/08/04 08:26:18 tg Exp $
--- security/nss/lib/freebl/Makefile.orig	Thu Jun 11 00:55:19 2009
+++ security/nss/lib/freebl/Makefile	Sun Mar 14 17:46:07 2010
@@ -224,7 +224,7 @@ endif
 # to bind the blapi function references in FREEBLVector vector
 # (ldvector.c) to the blapi functions defined in the freebl
 # shared libraries.
-ifeq (,$(filter-out BSD_OS FreeBSD Linux NetBSD OpenBSD, $(OS_TARGET)))
+ifeq (,$(filter-out BSD_OS FreeBSD Linux NetBSD OpenBSD MidnightBSD MirBSD, $(OS_TARGET)))
     MKSHLIB += -Wl,-Bsymbolic
 endif
 
